sources/packages/backend/apps/api/src/route-controllers/designation-agreement/designation-agreement.aest.controller.ts

```diff
@@ -55,13 +56,13 @@ export class DesignationAgreementAESTController extends BaseController {
   /**
    * Retrieve the designation agreement information and
    * the associated locations approvals.
-   * @param designationId
+   * @param designationId id of the designation to be retrieved.
    * @returns designation agreement information.
    */
   @Get(":designationId")
   async getDesignationAgreement(
```

Comment:
@ApiNotFoundResponse ? `Not found exception is potentially thrown from this.designationAgreementControllerService.getDesignationAgreement`